digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectDeltaCalculator.scala

```diff
@@ -83,8 +95,24 @@ object ProjectDeltaCalculator {
     ).toSeq
   }
 
+  def partition(roadAddresses: Seq[RoadAddress], projectLinks : Seq[ProjectLink]): (Seq[RoadAddressSection],Seq[RoadAddressSection]) = {
```

Comment:
Scaladoc for intended functionality - "Calculates the transfer sections on a road part and maps them together for change table"